### PR TITLE
FIXMEs in mongodb module

### PR DESCRIPTION
**Pull request by djsd123**
Replacing fixmes with documentation

"mongodb/templates/configure-node-priority.js.erb" Also has a FIXME, which will take me a little longer sort.

### CODE REVIEW

File: modules/mongodb/manifests/package.pp [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+# == Class: mongodb::package
+#
+# Define which mongodb package to install
+#
 class mongodb::package(
   $version,
   $package_name = 'mongodb-10gen',
```
Comment (surminus) [inline modules/mongodb/manifests/package.pp]: These parameters should be documented

**Comment by surminus** [comment]
Super!

File: modules/mongodb/manifests/monitoring.pp [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+# == Class: mongodb::monitoring
+#
+# Configures monitoring for mongodb
+#
 class mongodb::monitoring ($dbpath = '/var/lib/mongodb') {
```
Comment (deanwilson) [inline modules/mongodb/manifests/monitoring.pp]: This should be reformatted to a new line and have a `Parameter` section explaining it.

File: modules/mongodb/manifests/python.pp [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+# == Class: mongodb::python
+#
+# Installs PyMongo package
```
Comment (deanwilson) [inline modules/mongodb/manifests/python.pp]: This should be "Manages the python mongo client" so we can change the underlying pip package as needed.

File: modules/mongodb/manifests/s3backup/package.pp [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# == Class: mongodb::s3backup::package
+#
+# Installs third party packages on mongodb instances
```
Comment (deanwilson) [inline modules/mongodb/manifests/s3backup/package.pp]: `Installs packages required for MongoDB S3 backups`

**Comment by surminus** [comment]
:+1: fftm

**Pull request merged**